kornia/augmentation/_2d/geometric/translate.py

```diff
@@ -0,0 +1,108 @@
+            is randomly sampled in the range img_width * a < dx < img_width * b
+        translate_y: tuple of maximum absolute fraction for vertical translations.
+            For example translate_y=(a, b), then vertical shift
+            is randomly sampled in the range img_height * a < dy < img_height * b.
+        resample: resample mode from "nearest" (0) or "bilinear" (1).
+        padding_mode: padding mode from "zeros" (0), "border" (1) or "refection" (2).
+        same_on_batch: apply the same transformation across the batch.
+        align_corners: interpolation flag.
+        p: probability of applying the transformation.
+        keepdim: whether to keep the output shape the same as input (True) or broadcast it to the batch form (False).
+
+    Shape:
+        - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`
+        - Output: :math:`(B, C, H, W)`
+
+    .. note::
+        This function internally uses :func:`kornia.geometry.transform.warp_affine`.
+
+    Examples:
+        >>> import torch
+        >>> rng = torch.manual_seed(0)
+        >>> input = torch.rand(1, 1, 3, 3)
+        >>> aug = RandomTranslate((0.2, 0.1), p=1.)
+        >>> out = aug(input)
+        >>> out, aug.transform_matrix
+        (tensor([[[[0.0050, 0.0523, 0.0700],
+                  [0.0460, 0.4861, 0.6642],
+                  [0.0168, 0.1877, 0.3913]]]]), tensor([[[1.0000, 0.0000, 0.6000],
+                 [0.0000, 1.0000, 0.6000],
+                 [0.0000, 0.0000, 1.0000]]]))
+        >>> aug.inverse(out)
+        tensor([[[[0.4026, 0.5886, 0.0605],
+                  [0.1977, 0.3985, 0.0419],
+                  [0.0171, 0.0371, 0.0039]]]])
+
+    To apply the exact augmenation again, you may take the advantage of the previous parameter state:
+        >>> input = torch.randn(1, 3, 32, 32)
+        >>> aug = RandomTranslate((0.2, 0.1), p=1.)
+        >>> (aug(input) == aug(input, params=aug._params)).all()
+        tensor(True)
+    """
+
+    def __init__(
+        self,
+        translate_x: Optional[Union[Tensor, Tuple[float, float]]] = None,
+        translate_y: Optional[Union[Tensor, Tuple[float, float]]] = None,
+        resample: Union[str, int, Resample] = Resample.BILINEAR.name,
+        same_on_batch: bool = False,
+        align_corners: bool = False,
+        padding_mode: Union[str, int, SamplePadding] = SamplePadding.ZEROS.name,
+        p: float = 0.5,
+        keepdim: bool = False,
+    ) -> None:
+        super().__init__(p=p, same_on_batch=same_on_batch, keepdim=keepdim)
+        self._param_generator: rg.TranslateGenerator = rg.TranslateGenerator(translate_x, translate_y)
+        self.flags = dict(
+            resample=Resample.get(resample), padding_mode=SamplePadding.get(padding_mode), align_corners=align_corners
+        )
+
+    def compute_transformation(self, input: Tensor, params: Dict[str, Tensor], flags: Dict[str, Any]) -> Tensor:
+        translations = torch.stack([params["translate_x"], params["translate_y"]], dim=-1)
```

Comment:
```suggestion
        translations = stack([params["translate_x"], params["translate_y"]], dim=-1)
```
we have exposed stack to kornia.core